packages/microcosm-www-next/src/components/header.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+
+export default class Header extends React.Component {
+  componentDidMount() {
+    this.el = document.getElementById('fade-in-header')
+    this.listenForAnimationEnd(this.el)
+  }
+
+  listenForAnimationEnd(el) {
+    el.addEventListener('animationend', () => {
+      el.classList.remove('fade-in')
+    })
+  }
+
+  componentWillUpdate() {
+    this.el.classList.add('fade-in')
```

Comment:
when the header updates with new data, it will add `.fade-in` which triggers an animation. when that animation ends, it will trigger the event listener on line 10 to remove `.fade-in`